src/test/java/blackjack/domain/game/BettingZoneTest.java

```diff
@@ -0,0 +1,44 @@
+package blackjack.domain.game;
+
+import blackjack.domain.player.Gambler;
+import blackjack.domain.player.Player;
+import blackjack.domain.player.Result;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+@SuppressWarnings("NonAsciiCharacters")
+public class BettingZoneTest {
+
+    @ParameterizedTest
+    @CsvSource({"BLACKJACK, 15000", "LOSE, -10000", "PUSH, 0", "WIN, 10000"})
+    void 겜블러의_수익을_계산한다(final Result result, final int expectedProfit) {
+        final BettingZone bettingZone = generateBettingZone();
+        final Map<Player, Result> resultByPlayers = generateResultByPlayers(result);
+
+        final Map<Player, Money> profitByPlayers = bettingZone.calculateProfitByPlayers(resultByPlayers);
+
+        assertThat(profitByPlayers.values())
+                .extracting(Money::getAmount)
+                .containsExactly(expectedProfit);
+    }
+
+    private BettingZone generateBettingZone() {
+        Map<Player, Money> betMoneyByPlayers = new HashMap<>();
+        betMoneyByPlayers.put(new Gambler("후추"), Money.createMoneyForBetting(10000));
+        return new BettingZone(betMoneyByPlayers);
```

Comment:
Map.of로 간단하게 처리할 수도 있을 것 같아요!

```suggestion
        return new BettingZone(Map.of(new Gambler("후추"), Money.createMoneyForBetting(10000)));
```

Reply:
와 훨씬 간단해진 것 같습니다. Map.of() 도 기억해두겠습니다! 감사합니다😁😁😁